models/service.go

```diff
@@ -33,6 +33,8 @@ type ServiceOverview struct {
 	Labels map[string]string `json:"labels"`
 	// Selector for Service
 	Selector map[string]string `json:"selector"`
+	// ExportTo for Service
+	ExportTo map[string]bool `json:"ExportTo,omitempty"`
```

Comment:
The json map should start with minor case "exportTo" to be similar as other mappings.
I.e.:
```
IstioSidecar bool `json:"istioSidecar"`
AdditionalDetailSample *AdditionalItem `json:"additionalDetailSample"`
IstioReferences []*IstioValidationKey `json:"istioReferences"`
```

Also, this is related to the discussion in the business/service.go, if we may want to propagated in all services, or just use it for Istio Registry use cases, then it would be empty in rest of the cases (it's fine with me if we don't need it).

If want to render it in the UI, then we'd need to adapt the buildService() list and also propagate this field in the UI type and ServiceDetails page I think.

Just a note, I think that this comment can be a side task not directly related with the PR, just leaving this thought to not forget.